test/app.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
-describe('App', () => {
-    it('should boot', (done) => {
-        done();
-    });
+var test = require('tape');
```

Comment:
I think we can safely use `const` and `let` and never use `var` again.

http://programmers.stackexchange.com/a/274352/145069
https://mathiasbynens.be/notes/es6-const